src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -12,19 +1,60 @@
+type Props = {
+  setStatus: (status: () => string) => void;
+  setQuery: (status: () => string) => void;
```

Comment:
The `setQuery` function should accept a string directly, not a function returning a string. Change the type to `(query: string) => void;`.